lib/edoc/doc/overview.edoc

```diff
@@ -377,6 +377,16 @@ The following tags can be used before a function definition:
       Useful for debug/test functions, etc. The content can be
       used as a comment; it is ignored by EDoc.</dd>
 
+  <dt><a name="ftag-private">`@returns'</a></dt>
```

Comment:
Good catch. Yes, the anchors need to be updated to match; that's a copy/paste error.